### PR TITLE
Fix code highlighting

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by kevmoo** [comment]
CC @isoos

**Comment by kevmoo** [comment]
Deployed at https://kevmoo-better-highlighting-dot-dartlang-pub.appspot.com/packages/built_mirrors

**Comment by mkustermann** [comment]
Seems like it's breaking more lines now. Maybe you could update it so it fits 80 characters on one line inside the `<code>` block?

Otherwise looks fine.

File: app/lib/templates.dart [hunk 1]
```diff
@@ -145,11 +160,7 @@ class TemplateService {
       final content = file.text;
       if (content != null) {
         if (isMarkdownFile(filename)) {
-          try {
-            return cmd.markdownToHtmlWithSyntax(content);
-          } catch (e) {
-            return cmd.markdownToHtmlWithoutSyntax(content);
-          }
+          return _markdownToHtmlWithoutSyntax(content);
```
Comment (kevmoo) [inline app/lib/templates.dart]: Well, just call it '_markdownToHtml` – since this method does no syntax highlighting. That happens on the client w/ JS

**Comment by kevmoo** [comment]
@mkustermann LGTY?

**Pull request merged**